### PR TITLE
Adds privacy policy overlay to about section & removes .html

### PR DESCRIPTION
**Pull request by simaonogueira101**
Creates an overlay with lorem ipsum text as a placeholder for the privacy policy info.

### CODE REVIEW

**Comment by MGago** [comment]
@simaonogueira101 can we have a link to go directly to the privacy policy? I think that Play stores demands it. Also, the Privacy policy is in the short-stories slack channel, can you add the real thing in this PR before merging?

**Comment by simaonogueira101** [comment]
My bad, I thought you wanted something like this. Yes, absolutely, I'm on it.

**Comment by simaonogueira101** [comment]
@MGago Done with those changes.

**Comment by simaonogueira101** [comment]
Please review this last part as well @MGago

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```
Comment (bffcorreia) [inline npm-debug.log]: Is this file needed in git? If not, please add it to git ignore and remove it from the repo :)

**Review by bffcorreia — APPROVED** [review]
Everything looks fine @simaonogueira101. Please just check my comment before merge :)
